controller/controllers/kalm_ns_controller.go

```diff
@@ -52,14 +53,22 @@ func NewKalmNSReconciler(mgr ctrl.Manager) *KalmNSReconciler {
 	}
 }
 
+// resource change needs trigger reconcile
+type MapAll struct{}
+
+func (m MapAll) Map(_ handler.MapObject) []reconcile.Request {
+	return []reconcile.Request{{NamespacedName: types.NamespacedName{}}}
+}
+
 func (r *KalmNSReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&v1.Namespace{}).
-		//For(&v1alpha1.HttpsCertIssuer{}).
-		//For(&v1alpha1.HttpsCert{}).
-		Watches(&source.Kind{Type: &v1alpha1.HttpsCertIssuer{}}, &handler.EnqueueRequestForObject{}).
-		Watches(&source.Kind{Type: &v1alpha1.HttpsCert{}}, &handler.EnqueueRequestForObject{}).
-		//Owns(&v1alpha1.HttpsCert{}).
+		Watches(&source.Kind{Type: &v1alpha1.HttpsCertIssuer{}}, &handler.EnqueueRequestsFromMapFunc{
```

Comment:
I suggest make a func that returns a source.Kind pointer. Params is type and map function

Reply:
updated